### ISSUE TITLE
Circle Annotation without Image captions 

### ISSUE THREAD

**Issue by kgschoueri** [Smithsonian/dpo-voyager#235]
With the 'extended' annotation style the caption shows up but this is not the case with the 'circle' style annotation. 
<img width="841" alt="Circle_normal_photo_no_caption" src="https://github.com/Smithsonian/dpo-voyager/assets/107149450/de334b31-235c-45c3-a89d-18b8e98366b9">

**Comment by gjcope** [comment]
Alt text is also missing in this scenario.

**Comment by gjcope** [comment]
Fixed in v0.36.0
